index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+      tailwind.config = {
+        theme: { extend: { colors: { clifford: "#da373d" } } },
+      };
+    </script>
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
```

Comment:
I think it would be better to use semantic tags <header>

Reply:
- I think you did a great job using the inline comments. :clap: :clap: However please remember to always include an explanation, a "why", for each requested change. Some things could be obvious for you but not for someone else :smiley: :smiley:
- [OPTIONAL] I think it's a good idea to use some emojis to make your review more human-friendly. :space_invader: :space_invader:

Reply:
- [ ] As mentioned by the previous reviewer, providing a reason for the change will help the author of the PR to quickly understand and identify :bulb:  the change/bug or improvement to be resolved in their code. 

In the case of this comment, please make sure to elaborate after you use "because"  :heavy_check_mark: 
> I think it would be better to use semantic tags `<header>` because it is for the header section. blushblushblushblushblush For reference: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md

Reply:
Is that okay? Do I need to add something?